app/controllers/spree/webhook/payways_controller.rb

```diff
@@ -42,8 +42,8 @@ def return_callback_handler(handler_service)
         end
 
         # the callback invoke by PAYWAY in case of success
-        payload = JSON.parse(params[:response])
```

Comment:
Don't change this. Legacy apps are still using this. You might check here: https://github.com/bookmebus/operator-tenant#payment-merchant-profile-settings. Let add this note to README as well.